src/webcore/value.rs

```diff
@@ -51,6 +51,16 @@ impl Reference {
     }
 }
 
+impl PartialEq for Reference {
+    #[inline]
+    fn eq( &self, other: &Reference ) -> bool {
+        js!( return @{self} === @{other}; ).try_into().unwrap()
```

Comment:
Hm... I remember putting a comment regarding this, but I can't find it now...? Did it disappear or was it not posted in the first place?

Reply:
You have to click on the "Show outdated" thingy. That happened because I changed the code, so GitHub "helpfully" assumes that your old comments are now outdated because of the code changes.

Reply:
Ah right, I didn't notice the second outdated!